### PR TITLE
📖 Update Readme

### PR DESCRIPTION
**Pull request by saqibameen**
Update the readme for more engaging content, more info, and elaborated setting up steps.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,152 +2,310 @@
...
-    <img src="https://cdn.jsdelivr.net/gh/storybookjs/brand@master/badge/badge-storybook.svg" alt="Storybook">
+    <img src="https://img.shields.io/badge/%20-storybook-502ab0?logo=storybook&logoColor=white" alt="Storybook">
   </a>
+
+</p>
+
+
+
+
+## 🧶 Daily
```
Comment (idoshamun) [inline README.md]: What is this emoji? It's not clear to me :confused:

File: README.md [hunk 2]
```diff
@@ -2,152 +2,310 @@
...
-## Don't know Daily!?
+We, as developers, spend a lot of time looking for valuable articles and blog posts. It's hard to catch up with all the latest happenings — coz it's frustrating and consumes a ton of time.
 
-Daily is a dev news curator delivering relevant updates to your new tab. It gathers and ranks articles from tens of different sources to help you optimize your time and boost your knowledge.
+That's why built Daily to help you:
 
+* 👨‍💻 Stay up-to-date
+* ⏳Save time
+* 📱Read anywhere
+* 🔖 Read when you want
```
Comment (idoshamun) [inline README.md]: I would mention something about the passive consumption of content. This is one of our unique points, as a user you just have to open a new tab. You don't need to "want reading news".

File: README.md [hunk 3]
```diff
@@ -2,152 +2,310 @@
...
 
-* [daily](https://github.com/dailynowco/daily) - This is the repository you are currently at and it is actually the repository with overall documentation, community ideas, suggestions and whatnot.
+### 🎨 Frontend
 
-### Frontend
+* [daily-apps](https://github.com/dailynowco/daily-apps) - Monorepo with all the frontend related projects since Daily 2.0 — Vue components library, API encapsulation library, Daily extension and everything related to frontend.
 
-* [daily-apps](https://github.com/dailynowco/daily-apps) - Monorepo with all the frontend related projects since Daily 2.0, our Vue components library, API encapsulation library, obviously Daily extension and others.
-* [daily-go](https://github.com/dailynowco/daily-go) - Our Daily Go webapp, with a story like interface (called toilet mode) and user's bookmarks.
+* [daily-go](https://github.com/dailynowco/daily-go) - Progressive web app (PWA) called **Daily Go** for mobile devices.  It comes with a story like interface, called **toilet mode**. You can also manage
```
Comment (idoshamun) [inline README.md]: The last sentence is cut in the middle

File: README.md [hunk 4]
```diff
@@ -2,152 +2,310 @@
...
+
+# npx: installed 160 in 10.92s     // Expected result
+# Using environment: development
+# Ran 1 seed files
+```
+
+
+That's it! 🥂
+
+Now you have all the required services running. Each project' repo explains what services are needed and how to get started with them.
```
Comment (idoshamun) [inline README.md]:
```suggestion
Now you have all the required services running. Each project's repo explains what services are needed and how to get started with them.
```

File: README.md [hunk 5]
```diff
@@ -2,152 +2,310 @@
...
+## 🎩 Core Team
+
+Meet the core team of Daily.
+* [@idoshamun](https://twitter.com/idoshamun)
+* [@tsahimatsliah](https://twitter.com/TsahiMatsliah)
+* [@saqibameen](https://twitter.com/saqibameen)
+
+Feel free to reach us out and say hi 👋.
+
+## 💬 What You Think of Daily?
```
Comment (idoshamun) [inline README.md]: What Do You Think of Daily?

**Comment by saqibameen** [comment]
⚡️ @idoshamun glad you liked the updated readme. I have made the requested changes. Take a look now!

**Pull request merged**